pyproject.toml

```diff
@@ -6,9 +125,89 @@ requires = ["setuptools",
             "extension-helpers"]
 build-backend = 'setuptools.build_meta'
 
+[tool.setuptools]
+include-package-data = true
+
+[tool.setuptools.packages.find]
+include = ["astropy*"]
+exclude = ["astropy._dev*"]
+namespaces = false
+
+[tool.setuptools.package-data]
+"*" = [
+    "data/*",
+    "data/*/*",
+    "data/*/*/*",
+    "data/*/*/*/*",
+    "data/*/*/*/*/*",
+    "data/*/*/*/*/*/*",
+]
+'astropy' = ['astropy.cfg', 'CITATION']
```

Comment:
Why use single quotes here and not double quotes? The latter is more consistent with the rest of the file.

Reply:
Because I wasn't consistent. I can setup an auto formatter for it?

Reply:
Hmm I am surprised I caught this before that pre-commit thingy.

Reply:
The pre-commit doesn't check this file for that sort of thing

Reply:
We can set up repo checker made by Scientific Python. For a different PR of course.

Reply:
Yes this is my next thing to do. I worked a while ago with them to get the pre-commit hook functioning properly, but it only checks the pyproject.toml for project configuration settings.